### ISSUE TITLE
Change Request: allow to ignore suppressions file

### ISSUE THREAD

**Issue by arthaud-proust** [eslint/eslint#20358]
### ESLint version

v9.39.1

### What problem do you want to solve?

ESLint currently uses the default (`eslint-suppressions.json`) or provided suppressions file to ignore violations.

[See the reproduction of the problem](https://github.com/arthaud-proust/eslint-repro)

## Example

file.ts :
```ts
const unusedVar = "this var is unused, so eslint will raise a problem"
```

eslint-suppressions.json :
```json
{
  "file.ts": {
   "@typescript-eslint/no-unused-vars": {
      "count": 1
    },
  },
```

Running `eslint .` will show no problem

### What do you think is the correct solution?

The ideal solution would be to have a way to tell eslint to ignore the suppressions file.

For example, a flag like:

```bash
yarn eslint --ignore-suppressions
```

That would make eslint behave as if there was no suppressions file at all, showing all the problems in the codebase.

### Participation

- [x] I am willing to submit a pull request for this change.

**Comment by DMartens** [comment]
Allowing to temporarily ignore the suppression file does not seem to be discussed in its [RFC](
https://github.com/eslint/rfcs/pull/119).
I don't understand the motivation behind this as you can see which files have suppressed rules in the suppression file.
Or do you want that ESLint prints a warning if there is at least one suppressed problem, e.g.
"Found no problems but currently X problems are suppressed".
Can you please update the issue with the motivation for this feature?

**Comment by arthaud-proust** [comment]
Hello @DMartens, your question brought me out of my tunnel vision, thanks. 

Our use case was to lint only modified files, but the eslint-suppression.json file we've made was making eslint ignore lot of existing errors.
We simply need to delete the suppression file because it isn't useful in our strategy.

Thank you for working on eslint !
